build_tools/scripts/bazel_to_cmake.py

```diff
@@ -167,7 +200,8 @@ def _convert_deps_block(self, **kwargs):
     #    package2::target
     deps = kwargs.get("deps")
     deps_list = [self._convert_target(dep) for dep in deps]
-    deps_list = sorted(list(set(deps_list)))  # Remove duplicates and sort.
+    #deps_list = sorted(list(set(deps_list)))  # Remove duplicates and sort.
+    deps_list = list(filter(None,OrderedDict(zip(deps_list, repeat(None)))))  # Remove duplicates, but preserve order of deps.
```

Comment:
nit: can you trim this line to 80 characters, by putting the comment on a separate line?

https://www.python.org/dev/peps/pep-0008/#maximum-line-length
http://google.github.io/styleguide/pyguide.html#32-line-length

Reply:
Thanks for catching this. Done.